### PR TITLE
latest cft and config changes for Pcluster

### PR DESCRIPTION
**Pull request by pawanrlelab**

### CODE REVIEW

File: dump/standardcatalogitems.json [hunk 1]
```diff
@@ -661,5 +661,96 @@
...
+                                            "ap-south-1" : "ami-0487808e91e9cb859"
+                                    },
+                                    {
+                                            "ap-southeast-1" : "ami-0387151f1797dfbc9"
+                                    },
+                                    {
+                                            "ap-southeast-2" : "ami-049b783c5aa1ed5f5"
+                                    }
+                            ],
+                            "ami_path" : "/RL/RG/StandardCatalog/ParallelCluster"
```
Comment (manikandan-thangavelu-rl) [inline dump/standardcatalogitems.json:747]: Change this to
/RL/RG/StandardCatalog/ParallelCluster-linux-ami

File: config/access-policy.json [hunk 2]
```diff
@@ -106,7 +106,9 @@
         "imagebuilder:GetInfrastructureConfiguration",
         "imagebuilder:TagResource",
         "imagebuilder:UntagResource",
-        "cognito-idp:DeleteUserPool"
+        "cognito-idp:DeleteUserPool",
+        "codebuild:DeleteProject",
+        "route53:*"
```
Comment (manikandan-thangavelu-rl) [inline config/access-policy.json:111]: There was a iam:ListPolicyVersion permission needed right?

**Review by manikandan-thangavelu-rl — CHANGES_REQUESTED** [review]

File: config/access-policy.json (hunk 2 above)
Comment (pawanrlelab) [reply config/access-policy.json:111]: The policy was already there on line no. 48 that's why didn't added that.

File: dump/standardcatalogitems.json (hunk 1 above)
Comment (pawanrlelab) [reply dump/standardcatalogitems.json]: made the suggested change

**Review by manikandan-thangavelu-rl — APPROVED** [review]

**Pull request merged**